sunbeam-python/sunbeam/provider/local/deployment.py

```diff
@@ -306,3 +306,7 @@ def get_space(self, network: Networks) -> str:
         Local deployment only supports management space as of now.
         """
         return "management"
+
+    def get_spaces(self, networks: list[Networks]) -> set:
```

Comment:
```suggestion
    def get_spaces(self, networks: list[Networks]) -> set[str]:
```

Reply:
This code chunk is removed in the subsequent push